docs/sources/reference/run.md

```diff
@@ -451,6 +451,19 @@ familiar with using LXC directly.
 > you can use `--lxc-conf` to set a container's IP address, but this will not be
 > reflected in the `/etc/hosts` file.
 
+## Logging drivers (--log-driver)
+
+You can specify a different logging driver for the container than for the daemon.
+
+### Logging driver: none
+
+Disables any logging for the container.
```

Comment:
I would add that `docker logs` is unavailable with this - as this is the mode that changes the norm - same above actually.